shared/style/confirm.css

```diff
@@ -50,6 +50,7 @@ form[role="dialog"][data-type="confirm"] h1 {
   color: #fff;
   margin: 0;
   padding: 1rem 1.5rem 0;
+  word-wrap: break-word;
```

Comment:
Mmm I'm not sure an ellipsis would be a good idea after all. Please keep it like this :)